src/Package/Impl/StatusBar/VsStatusBar.cs

```diff
@@ -45,10 +45,15 @@ internal sealed class VsStatusBar : IStatusBar {
         }
 
         #region IStatusBar
-        public IDisposable AddItem(UIElement element) {
+        public IDisposable AddItem(Func<UIElement> itemFactory, object dataContext) {
```

Comment:
Generally, workflow accumulated a bit too much stuff, it pulls in many things not really needed for a simple session.